modules/appeals_api/spec/requests/v1/notice_of_disagreements/evidence_submissions_controller_spec.rb

```diff
@@ -6,24 +6,51 @@
 describe AppealsApi::V1::DecisionReviews::NoticeOfDisagreements::EvidenceSubmissionsController, type: :request do
   let(:notice_of_disagreement) { create(:notice_of_disagreement) }
   let(:evidence_submissions) { create_list(:evidence_submission, 3, supportable: notice_of_disagreement) }
-  let(:path) { '/services/appeals/v1/decision_reviews/notice_of_disagreements/evidence_submissions/' }
+
+  def base_path(path)
+    "/services/appeals/v1/decision_reviews/notice_of_disagreements/evidence_submissions/#{path}"
+  end
 
   describe '#show' do
+    let(:path) { base_path notice_of_disagreement.id }
```

Comment:
I like this, nice!

Reply:
I can't take credit lol. It's all over the codebase ;)

Reply:
...and I took it out after changing upload to create lol